handler/src/main/java/io/netty/handler/logging/LoggingHandler.java

```diff
@@ -55,7 +55,16 @@
     public LoggingHandler() {
         this(DEFAULT_LEVEL);
     }
-
+    /**
+     * Creates a new instance whose logger name is the fully qualified class
+     * name of the instance.
+     *
+     * @param enableHexDump hexDump enable or not.
+     */
+    public LoggingHandler(boolean enableHexDump) {
```

Comment:
I would suggest following other constructors and take `ByteBufFormat` as an argument instead of `boolean`
```suggestion
    public LoggingHandler(ByteBufFormat format) {
```